linera-core/src/client/mod.rs

```diff
@@ -3766,14 +3758,21 @@ impl<Env: Environment> ChainClient<Env> {
         &self,
         senders: &mut HashMap<ValidatorPublicKey, AbortHandle>,
     ) -> Result<impl Future<Output = ()>, ChainClientError> {
-        let (chain_id, nodes, local_node) = {
-            let committee = self.local_committee().await?;
+        let (nodes, local_node) = {
+            let committee = match self.local_committee().await {
+                Ok(committee) => committee,
+                Err(LocalNodeError::BlobsNotFound(_)) => {
+                    self.synchronize_chain_state(self.chain_id).await?;
+                    self.local_committee().await?
+                }
+                Err(err) => return Err(err.into()),
```

Comment:
Would it make sense to have this fallback in `self.local_committee()`? Or is this the only case